modules/govuk_jenkins/templates/jobs/clear_cdn_cache.yaml.erb

```diff
@@ -0,0 +1,46 @@
+---
+- job:
+    name: clear-cdn-cache
+    display-name: Clear production CDN cache
+    project-type: freestyle
+    description: >
+      Clears the production CDN cache for the 10 most popular pages on GOV.UK.
+    properties:
+      - build-discarder:
+          days-to-keep: 30
```

Comment:
I think these should also use `num-to-keep`, and I would be tempted to make it slightly higher than 10, perhaps 30.

Reply:
Yeah I just noticed that myself.  Updated, and changed to 30.

I've also removed the CDN job entirely from non-prod environments since it's hard coded to production.